test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -137,16 +106,24 @@ func assertManagedStatus(
 	etcHostsContent := ""
 
 	for startTime := time.Now(); time.Since(startTime) < retryTimeout; {
-		etcHostsContent = config.getEtcHostsContent(podName, name)
-		isManaged := strings.Contains(etcHostsContent, etcHostsPartialContent)
-
-		if expectedIsManaged == isManaged {
-			return
+		etcHostsContent = config.getFileContents(podName, name, "/etc/hosts")
+		etcHostsOriginalContent := config.getFileContents(podName, name, "/etc/hosts-original")
```

Comment:
Shall we declare these two filepath as const variables for better maintenance?

Reply:
Done